cachito/workers/scm.py

```diff
@@ -136,14 +134,13 @@ def clone_and_archive(self):
 
             cmd = ['git', 'clone', '-q', '--no-checkout', self.url, clone_path]
             log.debug('Cloning the repo with "%s"', ' '.join(cmd))
-            git_clone = subprocess.Popen(cmd, stderr=subprocess.PIPE)
-            _, error_output = git_clone.communicate()
+            git_clone = subprocess.run(cmd, capture_output=True, universal_newlines=True,
+                                       encoding='utf-8')
```

Comment:
Optional: I'd prefer the following formatting:
```python
git_clone = subprocess.run(
    cmd, capture_output=True, universal_newlines=True, encoding='utf-8')
```

Reply:
As counterpoint, I tend to use Lui's formatting. :)